src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './carina';
```

Comment:
Because the consumer should not care about the internal structure of the module.